### PR TITLE
New Adapter: Oraki

### PR DESCRIPTION
**Pull request by BenOraki**
DOC
https://github.com/prebid/prebid.github.io/pull/5403

### CODE REVIEW

File: static/bidder-info/oraki.yaml
Comment (gargcreation1992) [inline static/bidder-info/oraki.yaml:3]: Sent email for verification.

**Comment by bsardo** [comment]
Hi @BenOraki, we noticed that your adapter is basically the same as the Playdigo and QT adapters that were recently merged into the project, as well as the Copper6SSP adapter that is currently under review. We would like to point out that aliasing an adapter may be an option for you and is preferable if you and another bidder share the same server. You can read more about aliasing here: [aliasing an adapter](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#aliasing-an-adapterhttps://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#aliasing-an-adapter).

Please let us know your thoughts on whether this is a possibility for you.

**Comment by BenOraki** [comment]
Hi @bsardo, we do not share the same server with any of the adapters you mentioned. So we cannot use aliases. This pull request is very time-sensitive, and we must get it out as soon as possible. However, I will consider some methods of code duplication reduction in the future for our adapter.

File: static/bidder-info/oraki.yaml
Comment (gargcreation1992) [reply static/bidder-info/oraki.yaml:3]: email verified.

File: static/bidder-info/oraki.yaml
Comment (gargcreation1992) [inline static/bidder-info/oraki.yaml:16]: usersync works!
```
https://sync.oraki.io/pbserver?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Doraki%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%5BUID%5D
```

**Review by gargcreation1992 — APPROVED** [review]

**Comment by BenOraki** [comment]
@gargcreation1992 I see that this PR has been approved. Can it be merged now?

**Review by przemkaczmarek — APPROVED** [review]

**Comment by bsardo** [comment]
Hi @BenOraki, we just got a second approval so your PR is ready to merge. I'm holding off on doing so at the moment until I fix a critical issue with the latest release and then we will merge this in.

**Comment by BenOraki** [comment]
Hi @bsardo, why wasn’t this PR included in the last release?

**Comment by bsardo** [comment]
@BenOraki we had a freeze on merging PRs for the last release due to the critical issue mentioned earlier. This is being merged now and will be part of v2.29.0.

**Pull request merged**
